tests/sentry/nodestore/bigtable/backend/tests.py

```diff
@@ -93,43 +93,6 @@ def test_get(ns, compression, expected_prefix):
     assert raw_data.startswith(expected_prefix)
 
 
-def test_get_multi(ns):
```

Comment:
These tests have moved into test_common where they are applied to both backends.